### PR TITLE
RTCIceServerStats relayProtocol

### PR DESCRIPTION
**Pull request by lennart-csio**
for issue #512 
clarifying the RTCIceServerStats protocol by renaming to relayProtocol and using similar text and naming than in RTCIceCandidateStats

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -4512,13 +4512,14 @@ <h2>
...
+                <dfn><code>relayProtocol</code></dfn> of type <span class=
                 "idlMemberType">DOMString</span>
               </dt>
               <dd>
                 <p>
-                  Valid values for transport is one of <code>udp</code> and <code>tcp</code>. Based
-                  on the "transport" defined in [RFC5245] section 15.1.
+                  It is the protocol used by the endpoint to communicate with the ICE server.
+                  Valid values are <code>udp</code>, <code>tcp</code>, or <code>tls</code>.
+                  This is the same value that is used for the relay protocol of local ICE candidates.
```
Comment (alvestrand) [inline webrtc-stats.html]: What is the reference for "the relay protocol of local ICE candidates"?
Do we have a field in [[WEBRTC]] for it? Or is it an ICE-defined property? If so, which RFC?

Comment (vr000m) [reply webrtc-stats.html]: @fippo could you weigh in...

**Comment by henbos** [comment]
@vr000m to rebase and add reference

**Comment by henbos** [comment]
Ping @vr000m

File: webrtc-stats.html (hunk 1 above)
Comment (fippo) [reply webrtc-stats.html]: missed that This is an internal reference to https://w3c.github.io/webrtc-stats/#dom-rtcicecandidatestats-relayprotocol

Comment (vr000m) [reply webrtc-stats.html]: Fixed

**Comment by vr000m** [comment]
@alvestrand @henbos Please take a look

**Pull request merged**